projectforum/projects/project_filters.py

```diff
@@ -59,11 +125,7 @@ def projects_JSON_response(projects):
             'amount': the_project.amount,
             'status': the_project.status,
             'tags': tags,
-            'timestamp': format_time(the_project.timestamp),
-            'team_members': convert_people_to_list(
-                                the_project.team_members.all()
-                            ),
-            'applicants': convert_people_to_list(the_project.applicants.all())
+            'timestamp': format_time(the_project.timestamp)
         })
     return HttpResponse(json.dumps(projects_json),
```

Comment:
JsonResponse does this exactly, I think. https://docs.djangoproject.com/en/1.8/ref/request-response/#jsonresponse-objects

I think either way, you are showing the json response to users, which is why it was important to not send sensitive stuff.